server.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.get('/', (req, res) => {
```

Comment:
assuming this isn't necessary and will get deleted as you fill the app out (we don't want to make a habit of keeping extraneous code.

Reply:
it is necessary actually!